financial-connections/src/main/java/com/stripe/android/financialconnections/features/partnerauth/PartnerAuthViewModel.kt

```diff
@@ -262,25 +267,25 @@ internal class PartnerAuthViewModel @Inject constructor(
                 )
             )
             requireNotNull(authSession)
-            val result = cancelAuthorizationSession(authSession.id)
-            if (authSession.isOAuth) {
-                // For OAuth institutions, create a new session and navigate to its nextPane (prepane).
-                logger.debug("Creating a new session for this OAuth institution")
-                // Send retry event as we're presenting the prepane again.
-                postAuthSessionEvent(authSession.id, AuthSessionEvent.Retry(Date()))
-                // for OAuth institutions, we remain on the pre-pane,
-                // but create a brand new auth session
-                setState { copy(authenticationStatus = Uninitialized) }
-                createAuthSession()
-            } else {
-                // For OAuth institutions, navigate to Session cancellation's next pane.
-                postAuthSessionEvent(authSession.id, AuthSessionEvent.Cancel(Date()))
-                navigationManager.navigate(
-                    NavigateToRoute(
-                        command = result.nextPane.toNavigationCommand(),
-                        popCurrentFromBackStack = true
+            if (manifest.enableRetrieveAuthSession()) {
+                // client did not receive any deeplink -> retrieve the auth session and check its status
```

Comment:
update this comment to something along the lines of, "if the client canceled mid-flow (either by closing the browser or cancelling on the institution page), retrieve the auth session and try to recover if possible"

Reply:
https://github.com/stripe/stripe-android/pull/7125/commits/9735a5a8012ee3a4ec600bae34f0d0227453f9f5